identity/src/main/java/com/stripe/android/identity/IdentityVerificationSheet.kt

```diff
@@ -92,5 +96,31 @@ interface IdentityVerificationSheet {
             identityVerificationCallback: IdentityVerificationCallback
         ): IdentityVerificationSheet =
             StripeIdentityVerificationSheet(from, configuration, identityVerificationCallback)
+
+        /**
+         * Creates a [IdentityVerificationSheet] instance in a [Composable]. Which would be
+         * recreated if [configuration] or [identityVerificationCallback] changed.
+         *
+         * This API registers an [ActivityResultLauncher] into LocalContext.current and notifies its
+         * result to [identityVerificationCallback].
+         */
+        @Composable
+        fun rememberIdentityVerificationSheet(
+            configuration: Configuration,
+            identityVerificationCallback: IdentityVerificationCallback
+        ): IdentityVerificationSheet {
+            val context = LocalContext.current
+            val activityResultLauncher = rememberLauncherForActivityResult(
+                IdentityVerificationSheetContract(),
+                identityVerificationCallback::onVerificationFlowResult
+            )
+            return remember(configuration, identityVerificationCallback) {
```

Comment:
This should remember only on `configuration`, not on `identityVerificationCallback`. `identityVerificationCallback` changes are already handled inside `rememberLauncherForActivityResult`.
I think if you keep it like this it will create a new `StripeIdentityVerificationSheet` for every recomposition, because the callback is recreated. You can verify that by putting a breakpoint on line 118 and seeing if it's triggered multiple times while using the example app.

Reply:
👍  Verified the callback would trigger recreating on each recompose. 

The `configuration` will be recreated on each recompose too, but it alone won't retrigger the sheet recompose because the newly recomposed value would be `==` to the old one, as they are data classes. Made it remember-ed from the call site to prevent it being recalculated on each recompose